pom.xml

```diff
@@ -32,11 +32,11 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <maven.compiler.source>11</maven.compiler.source>
-    <maven.compiler.target>11</maven.compiler.target>
-    <maven.compiler.release>11</maven.compiler.release>
+    <maven.compiler.source>17</maven.compiler.source>
+    <maven.compiler.target>17</maven.compiler.target>
+    <maven.compiler.release>17</maven.compiler.release>
     <maven.compiler.parameters>true</maven.compiler.parameters>
-    <quarkus.version>3.2.6.Final</quarkus.version>
+    <quarkus.version>3.8.1</quarkus.version>
```

Comment:
We need obviously to create a branch to support 3.2 before merging this PR

Reply:
it was for that purpose that I have created this other issue https://github.com/quarkiverse/quarkus-logging-splunk/issues/241